src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+import java.io.FileNotFoundException;
+
+/**
+ * Main class for Duke application.
+ */
+public class Duke {
+
+    private TaskList tasklist;
```

Comment:
You might want to use the camel case for the variable name so it becomes `taskList`. However, leaving it as so is fine and readable too 😄 !